editors/code/package.json

```diff
@@ -3362,6 +3370,22 @@
                 },
                 {
                     "command": "rust-analyzer.openWalkthrough"
+                },
+                {
+                    "command": "rust-analyzer.syntaxTreeReveal",
+                    "when": "false"
```

Comment:
Ah I guess this perma hides the command (separate category would still be good I think)